Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -594,7 +598,16 @@ void UpdatePadding()
 #endif
 		}
 
+		void UpdateAutoFitMode()
+		{
 #if __MOBILE__
+			Control.AdjustsFontSizeToFitWidth = Element.AutoFit;
```

Comment:
I actually [tried to implement this earlier](https://www.youtube.com/watch?v=3iZoyuFgnUU) and found out that you will also need to set the `Lines=1` and `LineBreakMode=UILineBreakMode.Clip`. Doing this will also impact the other properties that influence this behavior. Did you test this?

Reply:
No i didnt...
I made this changes based on documentation and this #7205 issue comments... But i cant run IOS project because i dont have a mac

Reply:
@jfversluis I`ve watched your [video](https://www.youtube.com/watch?v=3iZoyuFgnUU) and ive tried on android ([like my example on of top this PR](https://user-images.githubusercontent.com/19656249/66719143-26afbb00-edc2-11e9-8026-9ffa1661fe06.gif)) use without MaxLines/LineBreakMode values changes.
I'm using AppCompatTextview autofit at some [personal projects](https://github.com/felipebaltazar/CustomCharts) as well, without these properties. I think that max lines will set only user need's to fit width only, and the same to Linebreakmode.

I don run IOS tests for this specific situation

Reply:
@jfversluis  I'm thinking in another way for this feature...
Maybe we can use an enum to `AutoFitTextMode` where we will have these options:

`AutoFitMode.WidthOnly` => this set max lines to 1 and line break mode to "don't wrap" to fit only in one way (i need to test this before)

`AutoFitMode.Uniform`  => autofit mode that fills all bounds with the text (width and height)

`AutoFitMode.None` => Default value to disable autofit mode.

And your suggestion to add autofit granularity, min and max font size as parameters.

What do you think?

Reply:
Sounds great, but I think this is very much how Android works, right? Which is fine! But how easy will it be to implement this in iOS?